games-action/powermanga/powermanga-0.93.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A 2D arcade shoot-em-up with 41 levels and more than 2000 sprites"
+DESCRIPTION="Made by TLK Games."
+HOMEPAGE="http://linux.tlk.fr/games/Powermanga"
+REVISION="1"
+COPYRIGHT="TLK Games 1998-2019"
+LICENSE="GNU GPL v3"
+SOURCE_URI="https://github.com/brunonymous/powermanga/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="010987a3cb27a1f9388a212f637977692284b5616952efa1efae09d2464e9249"
+SOURCE_DIR="Powermanga-$portVersion"
+PATCHES="powermanga-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	powermanga$secondaryArchSuffix = $portVersion
+	app:Powermanga$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libSDL_1.2$secondaryArchSuffix
+	lib:libSDL_mixer_1.2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libpng16$secondaryArchSuffix
+	devel:libSDL_1.2$secondaryArchSuffix
+	devel:libSDL_mixer_1.2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	./bootstrap
+	runConfigure ./configure --prefix $prefix
```

Comment:
Does this even build? Check other recipes on how they use runConfigure (prefix shouldn't be needed)

Reply:
It works with both, so I'll leave `--prefix $prefix` out.